public/scripts/pubminer.js

```diff
@@ -0,0 +1,115 @@
+// pubminer.js
+//
+// JavaScript to support pubminer UI results page
+
+// expand the row and fill with content from the backend
+// code adapted from samples on PatternFly.org
+$.fn.addArrowClickHandler = function() {
+    this.click(function(event){
+        if(!$(event.target).is("button, a, input, .fa-ellipsis-v")){
+            $(this).find(".fa-angle-right").toggleClass("fa-angle-down")
+            .end().parent().toggleClass("list-view-pf-expand-active")
+            .find(".list-group-item-container").toggleClass("hidden");
+
+            if($(this).find(".fa-angle-down").length > 0) {
+                let expansionBody = $(this).closest("div.list-group-item").find("div.expansion-body");
+                let uidValue = $(this).closest("div.list-group-item").find("input.uid").val();
+                // Get the content from back end
+                $.get('/detail/' + uidValue, null, function(response) {
+                    if (response.error) {
+                        expansionBody.text(response.error);
+                    } else {
+                        expansionBody.html("");
+                        for (var key in response) {
+                            if (response.hasOwnProperty(key)) {
+                                expansionBody.append($("<strong>", {html: key}));
+                                expansionBody.append(" " + response[key]);
+                                expansionBody.append("<br/>")
+                            }
+                        }
+                    }
+                });
+            }
+        } else {
```

Comment:
minor: is this `else` needed?

Reply:
I believe that's the way the sample code from PatternFly was written. Clearly it's not doing anything, so we can get rid of it the next time we edit the file.